phpcs.xml

```diff
@@ -20,19 +21,17 @@
 			<property name="newlinesCountBetweenOpenTagAndDeclare" value="0"/>
 		</properties>
 	</rule>
-	<rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration">
-		<properties>
-			<property name="usefulAnnotations" type="array" value="
-				@dataProvider,
-				@requires
-			"/>
-			<property name="enableObjectTypeHint" value="false"/>
```

Comment:
This can be replaced for all the rules by settings `php_version` to config:
https://github.com/slevomat/coding-standard/blob/614af5a6538aa7c96c45521522b329f770105cab/SlevomatCodingStandard/Sniffs/TypeHints/ParameterTypeHintSniff.php#L92

Done in this file above: 
![image](https://user-images.githubusercontent.com/924196/107156828-e42fcb80-6980-11eb-82ea-f36b13e3dd65.png)